src/api/app/models/bs_request.rb

```diff
@@ -888,7 +888,8 @@ def auto_accept
         rescue PostRequestNoPermission
           change_state(newstate: 'revoked', comment: 'Permission problem')
         rescue APIError
```

Comment:
```suggestion
        rescue APIError => e
```
Missing error handler...